<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1293</width>
    <height>972</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>1251</width>
      <height>871</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_7">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_8">
         <item>
          <widget class="QLabel" name="sizelbl">
           <property name="text">
            <string>Size: </string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="verticeslbl">
           <property name="text">
            <string>#Vertices: </string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="cellslbl">
           <property name="text">
            <string>#Cells: </string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_12">
         <item>
          <widget class="QVTKOpenGLWidget" name="qvtkWidget" native="true">
           <property name="minimumSize">
            <size>
             <width>900</width>
             <height>600</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QListWidget" name="modelList"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QPushButton" name="modelColorBtn">
           <property name="text">
            <string>Model Color</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="backgroundColorBtn">
           <property name="text">
            <string>Background Color</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="resetCameraBtn">
           <property name="text">
            <string>Reset Camera</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Light Intensity</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSlider" name="lightIntensitySlide">
           <property name="minimum">
            <number>0</number>
           </property>
           <property name="maximum">
            <number>1000</number>
           </property>
           <property name="pageStep">
            <number>100</number>
           </property>
           <property name="value">
            <number>500</number>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QCheckBox" name="clipChk">
           <property name="text">
            <string>Clip Filter</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QCheckBox" name="shrinkChk">
           <property name="text">
            <string>Shrink Filter</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSlider" name="shrinkSlide">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="maximum">
            <number>100</number>
           </property>
           <property name="value">
            <number>80</number>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_11">
         <item>
          <widget class="QCheckBox" name="boundingBoxChk">
           <property name="text">
            <string>Bounding Box</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="QCheckBox" name="warpChk">
           <property name="text">
            <string>Warp</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_6">
             <item>
              <widget class="QLabel" name="label_2">
               <property name="text">
                <string>x</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QSlider" name="warpXslide">
               <property name="enabled">
                <bool>false</bool>
               </property>
               <property name="minimum">
                <number>-20</number>
               </property>
               <property name="maximum">
                <number>20</number>
               </property>
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_10">
             <item>
              <widget class="QLabel" name="label_4">
               <property name="text">
                <string>y</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QSlider" name="warpYslide">
               <property name="enabled">
                <bool>false</bool>
               </property>
               <property name="minimum">
                <number>-20</number>
               </property>
               <property name="maximum">
                <number>20</number>
               </property>
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_9">
             <item>
              <widget class="QLabel" name="label_3">
               <property name="text">
                <string>z</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QSlider" name="warpZslide">
               <property name="enabled">
                <bool>false</bool>
               </property>
               <property name="minimum">
                <number>-20</number>
               </property>
               <property name="maximum">
                <number>20</number>
               </property>
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1293</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionFileOpen"/>
    <addaction name="actionFileSave"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionFileOpen"/>
   <addaction name="actionFileSave"/>
   <addaction name="actionFileAdd"/>
   <addaction name="separator"/>
   <addaction name="actionHelp"/>
   <addaction name="actionPrint"/>
  </widget>
  <action name="actionFileOpen">
   <property name="icon">
    <iconset resource="Icons/icons.qrc">
     <normaloff>:/Icons/fileopen.svg</normaloff>:/Icons/fileopen.svg</iconset>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionFileSave">
   <property name="icon">
    <iconset resource="Icons/icons.qrc">
     <normaloff>:/Icons/filesave.svg</normaloff>:/Icons/filesave.svg</iconset>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionHelp">
   <property name="icon">
    <iconset resource="Icons/icons.qrc">
     <normaloff>:/Icons/help.svg</normaloff>:/Icons/help.svg</iconset>
   </property>
   <property name="text">
    <string>Help</string>
   </property>
  </action>
  <action name="actionPrint">
   <property name="icon">
    <iconset resource="Icons/icons.qrc">
     <normaloff>:/Icons/print.svg</normaloff>:/Icons/print.svg</iconset>
   </property>
   <property name="text">
    <string>Print</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+P</string>
   </property>
  </action>
  <action name="actionFileAdd">
   <property name="icon">
    <iconset resource="Icons/icons.qrc">
     <normaloff>:/Icons/fileadd.svg</normaloff>:/Icons/fileadd.svg</iconset>
   </property>
   <property name="text">
    <string>Add</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+O</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QVTKOpenGLWidget</class>
   <extends>QWidget</extends>
   <header>QVTKOpenGLWidget.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="Icons/icons.qrc"/>
 </resources>
 <connections/>
</ui>
